### ISSUE TITLE
search doesn't show all results

### ISSUE THREAD

**Issue by rgrinberg** [Anaconda-Platform/anaconda-client#383]
For example:

```
$ anaconda search r-dplyr
```

Will show me no results.

But there are obviously plenty here: https://anaconda.org/search?q=r-dplyr

**Comment by goanpeca** [comment]
Maybe we are searching on the channels you have added? anyway it is definitely a bug

**Comment by abarto** [comment]
This is a server issue related to the ppc architectures. It'll be fixed once the fix/ppc-issues in anaconda-server is merged.

**Comment by abarto** [comment]
The branch fix/ppc-issues was merged into develop.

**Comment by abarto** [comment]
@dsludwig This is still happening in 2.27.0.

**Comment by abarto** [comment]
My bad. It works just fine w/ 2.27.0. I recommend we close this issue once 2.27.0 reaches GA.

**Comment by abarto** [comment]
This was fixed with the 2.27.3 server release.
